src/linkify-matrix.ts

```diff
@@ -36,20 +36,22 @@ enum Type {
     GroupId = "groupid"
 }
 
+// Linkifyjs types don't have parser, which really makes this harder.
+const linkifyTokens = (linkifyjs as any).parser.TOKENS;
+enum MatrixLinkInitialToken {
+    POUND = linkifyTokens.POUND,
+    PLUS = linkifyTokens.PLUS,
+    AT = linkifyTokens.AT,
```

Comment:
We normally use PascalCasing for enum members but might be acceptable here given we're just doing this to make TS happy with what is effectively an enum in that library. Also we could probably just do a `/@types/linkifyjs.d.ts` which defines `parser.TOKENS` as a usable enum. Might not be worth the effort given the looming v3 upgrade.

Reply:
Yeah, those were literally pre-upgrade changes so it's not as massive of a change.